src/routes.ts

```diff
@@ -8,6 +8,7 @@ export const sitemap = '/sitemap-0.xml';
 export const githubUrl = 'https://github.com/crossplane/crossplane';
 export const slackUrl = 'https://slack.crossplane.io/';
 export const docsUrl = 'https://docs.crossplane.io';
+export const docsGetStartedUrl = 'https://docs.crossplane.io/v1.10/getting-started/';
```

Comment:
the link should go to `docs.crossplane.io/latest`. `/latest` always redirects to the latest version.